routes/api/assets/index.js

```diff
@@ -40,6 +40,20 @@ router.get('/', (req, res, next) => {
 
 });
 
+// get multiple assets with a comma-separated list of asset ids
+router.get('/multi', (req, res, next) => {
+  const assetIds = req.query.ids.split(',');
+
+  Asset.findMultipleById(assetIds)
+    .then(assets => {
+      res.json(assets);
+    })
+    .catch(error => {
+      error.status = 500;
+      next(error);
+    });
+});
+
```

Comment:
I thought it we had discussed that the hydration on the user comments endpoint would be better?